### ISSUE TITLE
How to generate a flipped svg file?

### ISSUE THREAD

**Issue by aimerneige** [notnil/chess#101]
Hello! I am trying to generate a flipped SVG file for the black player.

I write code like this:

```go
	board := pos.Board()
	if pos.Turn() == chess.Black {
		board = board.Flip(chess.UpDown).Flip(chess.LeftRight)
	}
	if err := image.SVG(f, board); err != nil {
		log.Fatal(err)
	}
```

But I got this, where the a1 should be at the left top

![image](https://user-images.githubusercontent.com/51701792/172196341-58b077bb-f4c7-4213-9502-7cc6d4f7cfb4.png)

What should I do to generate a correct SVG file? Or is this a bug?

**Comment by notnil** [comment]
Thanks for the visualization.  Yeah chess/image will likely need to be updated to support "from black" view.  Flipping flips the pieces, not the board.

**Comment by notnil** [comment]
Fixed in #109
